aiida/manage/tests/main.py

```diff
@@ -325,7 +325,7 @@ def create_profile(self):
             self.create_aiida_db()
 
         if not self.root_dir:
-            self.root_dir = tempfile.mkdtemp()
+            self.root_dir = tempfile.TemporaryDirectory().name  # pylint: disable=consider-using-with
```

Comment:
```suggestion
            temp_dir = tempfile.TemporaryDirectory()
            self.root_dir = temp_dir.name  # pylint: disable=consider-using-with
            temp_dir.cleanup()
```

Reply:
We cannot use it here, because it needs to be kept alive for the duration of the tests. The cleanup is called manually in `destroy_all` which is called at the end of the test suite.

Reply:
I see, found that. Thanks!